subroutines.sh

```diff
@@ -3,39 +3,71 @@
 # subroutines.sh
 #
 
+
 # Function
 send_command() {
   vin=$1
-  shift
+  cmd=$2
+  cmd_name=$3
+
+  TESLA_CONTROL_CMD='/usr/bin/tesla-control -ble -vin $vin -key-name /share/tesla_blemqtt/${vin}_private.pem -key-file /share/tesla_ble_mqtt/${vin}_private.pem $cmd 2>&1'
 
+  # add a retry loop
   max_retries=5
-  for count in $(seq $max_retries); do
-    log_notice "Sending command $* to vin $vin, attempt $count/${max_retries}"
+  for tesla_ctrl_count in $(seq $max_retries); do
```

Comment:
that will work without initializing tesla_ctrl_count? I assume yes, but you are the expert here

Reply:
Hmm had forgotten already about bashio. Right, that probably doesn't work. The reason I'm renaming those counts is in case I end up doing ~ multi-threading in bash using FIFO pipes for all background processes ... I actually started to do that for the upcoming BLE Presence.